tests/foreman/ui/test_repository.py

```diff
@@ -740,6 +741,41 @@ def test_positive_end_to_end_custom_ostree_crud(session, module_prod):
         assert not session.repository.search(module_prod.name, new_repo_name)
 
 
+def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
+    """Sync ansible collection repository from ansible gallaxy
+
+    :id: f3212dbd-3b8a-49ad-ba03-58f059150c04
+
+    :expectedresults: All content synced successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+
+    """
+    repo_name = f'gallaxy-{gen_string("alpha")}'
+    requirements = '''
+    ---
+    collections:
```

Comment:
do we want to have such constant in  [robottelo/constants/__init__.py](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/constants/__init__.py) ?

Reply:
If anything I'd recommend storing this as a file in a `data` directory and read it into a string in the test module. Considering this non-blocking.